regulations/templatetags/paragraphs.py

```diff
@@ -6,7 +6,10 @@
 
 @register.filter(name='pdepth')
 def pdepth(value):
-    depth = len(value.get("label", []) or []) - section_depth
-    if len(value.get("marker", []) or []) > 1:
-        depth = depth - (len(value.get("marker", []) or []) - 1)
-    return depth
+    if value.get("parent_type", "") == "appendix":
+        return 1
```

Comment:
This probably makes more sense as a default. What can be cleaned up in this function to basically say if there's no label etc.. return 1? I expect there's a bunch of unnecessary complexity present here.